services/121-service/package.json

```diff
@@ -13,8 +13,8 @@
     "lint": "eslint \"**/*.{ts,js}\"",
     "fix": "npm run lint -- --fix",
     "build": "nest build -p tsconfig.json",
-    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node dist/main.js",
-    "start:dev": "nodemon --watch src/**/*.json --exec npm run start:dev:nest",
+    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node --max-old-space-size=4096 dist/main.js",
+    "start:dev": "NODE_OPTIONS='--max-old-space-size=4096' nodemon --watch src/**/*.json --exec npm run start:dev:nest",
```

Comment:
Do you think it makes sense to put this in a separate PR? I am not 100% sure we should immediately release this to production before testing it on our test servers. And it is not needed for this immidiate fix. 

I read you can also put this in the .npmrc file  https://stackoverflow.com/a/75793324

Reply:
Yes, we can put it in `.npmrc` as well.
OK I think better to remove this part from this PR. Removing!